Cloud Application Manager/Troubleshooting/troubleshooting-tips.md

```diff
@@ -57,15 +58,38 @@ Windows instances deployed from the Cloud Application Manager Appliance (run the
 
 ```
 
+If problem persists connect to the instance over SSH or RDP and grab a copy of the agent /var/log/elasticbox/elasticbox-agent.log in the case of Linux or ProgramData\ElasticBox\elasticbox-agent.log in the case of Windows (please not that the folder is hidden in Windows and the exact path must be entered to navigate to the folder). Once the log file is collected please and [Contact support](mailto:incident@CenturyLink.com)
+
+**Solution 2**
+
+Make sure the communication from CAM Agent to CAM servers works correctly.
+
+1. Connect to the instance by SSH or RDP.
+
+2. Run a ping command to check basic connectivity:
+
+If using Cloud Application Manager:
+```
+ping -c3 cam.ctl.io
+```
+
+If using Cloud Application Manager Appliance:
+```
+ping -c3 10.0.0.1
 **Note:** Replace 10.0.0.1 with your appliance IP or hostname.
+```
+
+If the above command does not get a reply, please take the corrective actions inside the VM, routers and firewalls to allow unrestricted communication between agent and CAM servers or CAM appliance.
```

Comment:
I would also mention the option to configure a proxy for the agent, but I'm not sure how it could be configured manually (ask Roberto)